languoids/tree/indo1319/clas1257/germ1287/nort3152/west2793/high1289/high1286/midd1349/mode1258/baye1239/bava1246/cent1967/uppe1398/md.ini

```diff
@@ -8,3 +8,5 @@ macroareas =
 	Eurasia
 countries = 
 	Austria (AT)
+links =
+	[German (Upper Austrian)](https://wals.info/languoid/lect/wals_code_gau)
```

Comment:
Good find! I wasn't even aware we had this in WALS. The liks to WALS are not really curated here, though, but [harvested from WALS](https://github.com/glottolog/pyglottolog/blob/15e46bf283ba1100b46ae8f940f8e75cd85812d7/src/pyglottolog/links/util.py#L105-L111). So to make this change permanent, you'd have to file an issue at https://github.com/cldf-datasets/wals/issues